### PR TITLE
EN-13395: extended overridable config for other config files

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
## Reasoning behind the pull request
- previously, overridable config values could have only been used for `config.toml` values
  
## Proposed changes
- extend the functionality to work with other config files as well (`external.toml`, `p2p.toml`, `enableEpochs.toml` for now. Other files can be suggested by the reviewers)

## Testing procedure
- override config values for the indicated files

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: cmd/node/config/prefs.toml [hunk 1]
```diff
@@ -36,10 +36,12 @@
...
    # The Path indicates what value to change, while Value represents the new value represented as string. The node operator must make sure
    # to follow the same type of the original value (ex: uint32: "37", float32: "37.0", bool: "true")
+   # File represents the file name that holds the configuration. Currently, the supported files are: config.toml, external.toml, p2p.toml and enableEpochs.toml
    # -------------------------------
    # Un-comment and update the following section in order to enable config values overloading
    # -------------------------------
    # OverridableConfigTomlValues = [
-   #    { Path = "StoragePruning.NumEpochsToKeep", Value = "4" },
-   #    { Path = "MiniBlocksStorage.Cache.Name", Value = "MiniBlocksStorage" }
+   #    { Path = "StoragePruning.NumEpochsToKeep", Value = "4", File = "config.toml" },
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply cmd/node/config/prefs.toml]: done

**Review by iulianpascalau — DISMISSED** [review]

**Review by sstanculeanu — APPROVED** [review]

File: common/operationmodes/operationmodes.go [hunk 2]
```diff
@@ -9,38 +12,43 @@ const (
...
+// ParseOperationModes will check and parse the operation modes
+func ParseOperationModes(operationModeList string) ([]string, error) {
+	if len(operationModeList) == 0 {
+		return []string{}, nil
+	}
+
+	modes := strings.Split(operationModeList, ",")
 	if len(modes) == 0 {
-		return nil
+		return []string{}, nil
```
Comment (iulianpascalau) [inline common/operationmodes/operationmodes.go:23]: unreachable code. You can test to see I'm right :D

Comment (bogdan-rosianu) [reply common/operationmodes/operationmodes.go]: right. removed

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**